packages/ui/src/components/Banner/banner.tsx

```diff
@@ -84,28 +125,44 @@ export const Banner: React.FC<BannerProps> = ({
                     <Typography.Title
                         type={invertText ? 'inverted' : undefined}
                         level={titleLevel}
-                        ellipsis={{ rows: 3 }}
+                        ellipsis={{ rows: titleRows }}
                     >
                         {title}
                     </Typography.Title>
                     <Typography.Paragraph
-                        type={invertText ? 'inverted' : undefined}
-                        ellipsis={{ rows: 3 }}
+                        type={invertText ? 'inverted' : 'secondary'}
+                        ellipsis={{ rows: size === 'small' ? 2 : 3 }}
                         size={textSize}
                     >
                         {subtitle}
                     </Typography.Paragraph>
                 </div>
                 {Boolean(actionText) && (
-                    <Button
-                        type='primary'
-                        stateless
-                    >
-                        {actionText}
-                    </Button>
+                    size === 'small'
+                        ? (
+                            <div className={`${CLASS_PREFIX}-action-link`}>
+                                <Typography.Text
+                                    type={invertText ? 'inverted' : undefined}
+                                >
+                                    {actionText}
+                                </Typography.Text>
+                                <ChevronRight
+                                    color={invertText ? colors.white : 'currentcolor'}
```

Comment:
```suggestion
                                    color={invertText ? colors.white : colors.black}
```

Why currentColor is default?

Reply:
But default color of text is not current. It's primary....so it can lead to bug, if you somehow specify different color on some of div parents